app/configurator/components/tag.tsx

```diff
@@ -24,16 +24,21 @@ const TagTypography = styled(Typography)(({ theme }) => ({
   },
 }));
 
-const Tag = ({
-  children,
-  type,
-  ...props
-}: { children: ReactNode; type: TagType } & TypographyProps & {
-    component?: BoxProps["component"];
-  }) => {
+const Tag = React.forwardRef<
+  unknown,
```

Comment:
Because a `component` prop can be passed to `Tag` and then it wouldn't be span anymore 😅 This is the case in DatasetTags component, where `MUILink` is passed as `Tag`'s component. Maybe then it would be better to use HTMLElement instead of unknown?